### ISSUE TITLE
TypeError: undefined is not an object (evaluating '_react.PropTypes.string')

### ISSUE THREAD

**Issue by DomVinyard** [Hellenic/react-hexgrid#11]
Latest node and react, breaks with this error.

```
×
TypeError: undefined is not an object (evaluating '_react.PropTypes.string')
(anonymous function)
node_modules/react-hexgrid/lib/Hexagon/Text.js:102
   99 |   }(_react.Component);
  100 | 
  101 |   Text.propTypes = {
> 102 |     children: _react.PropTypes.string
  103 |   };
  104 |   exports.default = Text;
  105 | });
View compiled
(anonymous function)
node_modules/react-hexgrid/lib/Hexagon/Text.js:3
  1 | (function (global, factory) {
  2 |   if (typeof define === "function" && define.amd) {
> 3 |     define(["exports", "react"], factory);
  4 |   } else if (typeof exports !== "undefined") {
  5 |     factory(exports, require("react"));
  6 |   } else {
View compiled
./node_modules/react-hexgrid/lib/Hexagon/Text.js
node_modules/react-hexgrid/lib/Hexagon/Text.js:13
  10 |     factory(mod.exports, global.React);
  11 |     global.Text = mod.exports;
  12 |   }
> 13 | })(this, function (exports, _react) {
  14 |   "use strict";
  15 | 
  16 |   Object.defineProperty(exports, "__esModule", {
View compiled
__webpack_require__
/Users/dom/Desktop/bitopoly-client/webpack/bootstrap 9505ad37cf0f3551ccaa:669
  666 | };
  667 | 
  668 | // Execute the module function
> 669 | modules[moduleId].call(module.exports, module, module.exports, hotCreateRequire(moduleId));
  670 | 
  671 | // Flag the module as loaded
  672 | module.l = true;
View compiled
fn
/Users/dom/Desktop/bitopoly-client/webpack/bootstrap 9505ad37cf0f3551ccaa:87
  84 | 		console.warn("[HMR] unexpected require(" + request + ") from disposed module " + moduleId);
  85 | 		hotCurrentParents = [];
  86 | 	}
> 87 | 	return __webpack_require__(request);
  88 | };
  89 | var ObjectFactory = function ObjectFactory(name) {
  90 | 	return {
View compiled
(anonymous function)
node_modules/react-hexgrid/lib/index.js:3
  1 | (function (global, factory) {
  2 |   if (typeof define === "function" && define.amd) {
> 3 |     define(['exports', './GridGenerator', './HexGrid', './HexUtils', './Layout', './Path', './Pattern', './Hexagon/Hexagon', './Hexagon/Text', './models/Hex'], factory);
  4 |   } else if (typeof exports !== "undefined") {
  5 |     factory(exports, require('./GridGenerator'), require('./HexGrid'), require('./HexUtils'), require('./Layout'), require('./Path'), require('./Pattern'), require('./Hexagon/Hexagon'), require('./Hexagon/Text'), require('./models/Hex'));
  6 |   } else {
View compiled
./node_modules/react-hexgrid/lib/index.js
node_modules/react-hexgrid/lib/index.js:13
  10 |     factory(mod.exports, global.GridGenerator, global.HexGrid, global.HexUtils, global.Layout, global.Path, global.Pattern, global.Hexagon, global.Text, global.Hex);
  11 |     global.index = mod.exports;
  12 |   }
> 13 | })(this, function (exports, _GridGenerator, _HexGrid, _HexUtils, _Layout, _Path, _Pattern, _Hexagon, _Text, _Hex) {
  14 |   'use strict';
  15 | 
  16 |   Object.defineProperty(exports, "__esModule", {
View compiled
__webpack_require__
/Users/dom/Desktop/bitopoly-client/webpack/bootstrap 9505ad37cf0f3551ccaa:669
  666 | };
  667 | 
  668 | // Execute the module function
> 669 | modules[moduleId].call(module.exports, module, module.exports, hotCreateRequire(moduleId));
  670 | 
  671 | // Flag the module as loaded
  672 | module.l = true;
View compiled
fn
/Users/dom/Desktop/bitopoly-client/webpack/bootstrap 9505ad37cf0f3551ccaa:87
  84 | 		console.warn("[HMR] unexpected require(" + request + ") from disposed module " + moduleId);
  85 | 		hotCurrentParents = [];
  86 | 	}
> 87 | 	return __webpack_require__(request);
  88 | };
  89 | var ObjectFactory = function ObjectFactory(name) {
  90 | 	return {
View compiled
./src/App.js
http://localhost:3000/static/js/bundle.js:106290:91
__webpack_require__
/Users/dom/Desktop/bitopoly-client/webpack/bootstrap 9505ad37cf0f3551ccaa:669
  666 | };
  667 | 
  668 | // Execute the module function
> 669 | modules[moduleId].call(module.exports, module, module.exports, hotCreateRequire(moduleId));
  670 | 
  671 | // Flag the module as loaded
  672 | module.l = true;
View compiled
fn
/Users/dom/Desktop/bitopoly-client/webpack/bootstrap 9505ad37cf0f3551ccaa:87
  84 | 		console.warn("[HMR] unexpected require(" + request + ") from disposed module " + moduleId);
  85 | 		hotCurrentParents = [];
  86 | 	}
> 87 | 	return __webpack_require__(request);
  88 | };
  89 | var ObjectFactory = function ObjectFactory(name) {
  90 | 	return {
View compiled
./src/index.js
http://localhost:3000/static/js/bundle.js:106507:82
__webpack_require__
/Users/dom/Desktop/bitopoly-client/webpack/bootstrap 9505ad37cf0f3551ccaa:669
  666 | };
  667 | 
  668 | // Execute the module function
> 669 | modules[moduleId].call(module.exports, module, module.exports, hotCreateRequire(moduleId));
  670 | 
  671 | // Flag the module as loaded
  672 | module.l = true;
View compiled
fn
/Users/dom/Desktop/bitopoly-client/webpack/bootstrap 9505ad37cf0f3551ccaa:87
  84 | 		console.warn("[HMR] unexpected require(" + request + ") from disposed module " + moduleId);
  85 | 		hotCurrentParents = [];
  86 | 	}
> 87 | 	return __webpack_require__(request);
  88 | };
  89 | var ObjectFactory = function ObjectFactory(name) {
  90 | 	return {
View compiled
(anonymous function)
http://localhost:3000/static/js/bundle.js:106645:37
__webpack_require__
/Users/dom/Desktop/bitopoly-client/webpack/bootstrap 9505ad37cf0f3551ccaa:669
  666 | };
  667 | 
  668 | // Execute the module function
> 669 | modules[moduleId].call(module.exports, module, module.exports, hotCreateRequire(moduleId));
  670 | 
  671 | // Flag the module as loaded
  672 | module.l = true;
View compiled
(anonymous function)
/Users/dom/Desktop/bitopoly-client/webpack/bootstrap 9505ad37cf0f3551ccaa:715
  712 | __webpack_require__.h = function() { return hotCurrentHash; };
  713 | 
  714 | // Load entry module and return exports
> 715 | return hotCreateRequire(0)(__webpack_require__.s = 0);
  716 | 
  717 | 
  718 | 
View compiled
global code
http://localhost:3000/static/js/bundle.js:717:12
This screen is visible only in development. It will not appear if the app crashes in production.
Open your browser’s developer console to further inspect this error.
```

**Comment by DomVinyard** [comment]
If i downgrade node and node-don from 16.0.0 to 15.5.4 it works fine

**Comment by Hellenic** [comment]
Looks like some components are still incorrectly referencing PropTypes from React package. Will have a look at some point, PR also welcome :)

**Comment by Hellenic** [comment]
PR came in which fixed this issue. Published version 1.0.1 in npm with those changes
